pos_cash_move_reason/wizard/wizard_pos_move_reason.py

```diff
@@ -90,6 +90,20 @@ def apply(self):
         self.ensure_one()
         AccountBankStatementLine = self.env["account.bank.statement.line"]
         AccountBankStatementLine.create(self._prepare_statement_line())
+        if self.move_type == "income":
+            chatter_text = _("<p>Money in.</p>")
+        elif self.move_type == "expense":
+            chatter_text = _("<p>Money out.</p>")
+        chatter_text += _(
+            "<p>Move reason: %s</p><p>Reason: %s</p> <p>Amount: %s</p>"
+        ) % (
+            self.move_reason_id.name,
+            self.name,
+            self.amount,
```

Comment:
Nitpicking / not blocking : Could you format amount ? 

![image](https://user-images.githubusercontent.com/3407482/221535794-5521a59c-685f-47b7-b098-c75f62b76716.png)

There is a function ``format_lang``  here : https://github.com/odoo/odoo/blob/14.0/odoo/tools/misc.py#L1290
Something like that could do the job.


```suggestion
            formatLang(self.env, self.amount, currency_obj=self.journal_id.currency_id),
```

Reply:
ok